tests/baselines/reference/api/tsserverlibrary.d.ts

```diff
@@ -4707,7 +4708,7 @@ declare namespace ts {
         getBreakpointStatementAtPosition(fileName: string, position: number): TextSpan | undefined;
         getSignatureHelpItems(fileName: string, position: number, options: SignatureHelpItemsOptions | undefined): SignatureHelpItems | undefined;
         getRenameInfo(fileName: string, position: number, options?: RenameInfoOptions): RenameInfo;
-        findRenameLocations(fileName: string, position: number, findInStrings: boolean, findInComments: boolean): ReadonlyArray<RenameLocation> | undefined;
+        findRenameLocations(fileName: string, position: number, findInStrings: boolean, findInComments: boolean, providePrefixAndSuffixTextForRename: boolean): ReadonlyArray<RenameLocation> | undefined;
```

Comment:
>providePrefixAndSuffixTextForRename [](start = 113, length = 35)

Is this public?  If so, does it need to be optional?

Reply:
@sheetalkamat thoughts? i think it's fine as is, given that `findInStrings` and `findInComments` aren't optional, but I'm happy to make it optional.

Reply:
eh, i'll just make it optional.

Reply:
Made the flag optional